e2e-test-app/src/main/java/com/sap/ai/sdk/app/controllers/OpenAiController.java

```diff
@@ -69,15 +69,10 @@ public static ResponseEntity<ResponseBodyEmitter> streamChatCompletionDeltas() {
     final Runnable consumeStream =
         () -> {
           final var totalOutput = new OpenAiChatCompletionOutput();
-          // try-with-resources ensures the stream is closed
-          try (stream) {
-            stream
-                .peek(totalOutput::addDelta)
-                .forEach(delta -> send(emitter, delta.getDeltaContent()));
-          } finally {
-            send(emitter, "\n\n-----Total Output-----\n\n" + objectToJson(totalOutput));
-            emitter.complete();
-          }
+          stream
```

Comment:
as discussed in slack, let's keep the try with resources because emitter might throw

Reply:
My intention was to offer demo / powerpoint-ready code as part of the test app, but I'm also fine with reverting - and sticking to technically-best-practice with try-with-resources.